packages/contact-exchange/package.json

```diff
@@ -20,5 +20,11 @@
   "keywords": [],
   "author": "adorsys",
   "license": "ISC",
-  "description": ""
+  "description": "",
+  "dependencies": {
+    "@adorsys-gis/multiple-did-identities": "file:../packages",
+    "axios": "^1.7.7",
```

Comment:
- You should have `@adorsys-gis/multiple-did-identities` automatically.
- Why include axios into the project? Isn't fetch enough?

Reply:
I am using axios to make a request to the mediator, i didn't notice we're currently using fetch for that.
Thank you.